jdaviz/configs/cubeviz/plugins/moment_maps/moment_maps.py

```diff
@@ -126,7 +126,8 @@ def vue_calculate_moment(self, *args):
         label = "Moment {}: {}".format(n_moment, self._selected_data.label)
         fname_label = self._selected_data.label.replace("[", "_").replace("]", "")
         self.filename = "moment{}_{}.fits".format(n_moment, fname_label)
-        self.data_collection[label] = self.moment
```

Comment:
I changed this so it would be consistent with the rest of the plugins, let me know if it needs to be reverted for whatever reason.